src/platforms/common/performance/instrumentation/opentelemetry.mdx

```diff
@@ -118,6 +118,6 @@ You'll have to configure both OpenTelemetry and Sentry to see transactions in Se
 
 With Sentry’s OpenTelemetry SDK, an OpenTelemetry `Span` becomes a Sentry `Transaction` or `Span`. The first `Span` sent through the Sentry `SpanProcessor` is a `Transaction`, and any child `Span` gets attached to the first `Transaction` upon checking the parent `Span` context. This is true for the OpenTelemetry root `Span` and any top level `Span` in the system. For example, a request sent from frontend to backend will create an OpenTelemetry root `Span` with a corresponding Sentry `Transaction`. The backend request will create a new Sentry `Transaction` for the OpenTelemetry `Span`. The Sentry `Transaction` and `Span` are linked as a trace for navigation and error tracking purposes.
 
-## Filtering
+## Additional Configuration
 
-If you want to filter out transactions before sending them to Sentry, ( to get rid of health checks, for example), you can use <PlatformLink to="/configuration/options/#before-send-transaction"><PlatformIdentifier name="before-send-transaction" /></PlatformLink> to drop them. <PlatformLink to="/configuration/filtering/#using-platformidentifier-namebefore-send-transaction-">Here's an example</PlatformLink>.
+If you need more fine grained control over Sentry, take a look at the <PlatformLink to="/configuration/">Configuration page</PlatformLink>. In case you'd like to filter out transactions before sending them to Sentry, (to get rid of health checks, for example), you may find the <PlatformLink to="/configuration/filtering/#filtering-transaction-events">Filtering page</PlatformLink> helpful.
```

Comment:
```suggestion
If you need more fine grained control over Sentry, take a look at the <PlatformLink to="/configuration/">Configuration page</PlatformLink>. In case you'd like to filter out transactions before sending them to Sentry (to get rid of health checks, for example), you may find the <PlatformLink to="/configuration/filtering/#filtering-transaction-events">Filtering page</PlatformLink> helpful.
```